bullet_train-super_scaffolding/Gemfile

```diff
@@ -11,5 +11,7 @@ gem "sprockets-rails"
 gem "bullet_train"
 gem "bullet_train-api"
 
+gem "refine-rails", github: "clickfunnels2/refine-rails"
```

Comment:
@jagthedrummer yeah, it's private for now, but it's in the process of being open-sourced, so it can't be in the `.gemspec` yet.  There's a version out that you can see via `gem install refine-rails`.

Reply:
Realized we could put it into the `.gemspec` since there's a gem version out.